terratest/src/test/pega/pega-deployment-with-and-without-image-pull-secrets_test.go

```diff
@@ -0,0 +1,234 @@
+package pega
+
+import (
```

Comment:
Can we add a test case for `.Values.global.docker.registry` being nil? May be testing for one of templates (may be Pega deployments) would be good enough

Reply:
Added test case.